movai_core_shared/logger.py

```diff
@@ -131,103 +131,6 @@ def get_logger(logger_name: str):
         logger.propagate = False
         return logger
 
-    @staticmethod
-    def get_logs(limit=1000, offset=0, level=None, tags=None, message=None, from_=None, to_=None, pagination=False,
-                 services=None):
```

Comment:
don't delete this funcion
make it backwards compatible 
and keep the function and direct it to the new logic